lookout/style/typos/analyzer.py

```diff
@@ -107,6 +107,57 @@ def analyze(self, ptr_from: ReferencePointer, ptr_to: ReferencePointer,
                         comments.append(comment)
         return comments
 
+    @staticmethod
+    def reconstruct_identifier(tokenizer: TokenParser, pred_tokens: List[str], identifier: str) \
+            -> str:
+        """
+        Reconstruct identifier given predicted tokens and initial identifier.
+
+        :param tokenizer: tokenizer - instance of TokenParser.
+        :param pred_tokens: list of predicted tokens.
+        :param identifier: identifier.
+        :return: reconstructed identifier based on predicted tokens.
+        """
+        identifier_l = identifier.lower()
+        # setup required parameters
+        single_shot = tokenizer._single_shot
+        min_split_length = tokenizer._min_split_length
+        tokenizer._single_shot = True
+        tokenizer._min_split_length = 1
+        # sanity checking
+        initial_tokens = list(tokenizer.split(identifier))
+        err = "Number of predicted tokens (%s) not equal to the number of tokens in the " \
+              "identifier (%s) for identifier '%s', predicted_tokens '%s', tokens in identifier " \
+              "'%s'"
+        assert len(initial_tokens) == len(pred_tokens), \
```

Comment:
Yep, otherwise, this check will fail anyway: if `min_split_length` is different, they will not be equal